src/metaswitch/ellis/prov_tools/delete_user.py

```diff
@@ -57,6 +58,17 @@ def main():
     if not utils.check_connection():
         sys.exit(1)
 
+    if not args.no_prompt:
+        print 'Are you sure you wish to delete the following users: {}? [y/n]'.format(args.dns)
+        choice = raw_input().lower()
+
+        if choice != 'y':
+            if choice != 'n':
+                print "Invalid response. Aborting"
+            else:
+                print "Aborting"
```

Comment:
"Aborting" sounds a little aggressive, generally CLI tools go for "Exiting on user request" or similar so the user's action gets direct feedback.